.circleci/config.yml

```diff
@@ -339,3 +371,65 @@ jobs:
           at: /tmp
       - push_image:
           tag: "${CIRCLE_TAG}"
+  # Runs integration tests against a kubernetes cluster
+  eks-integration-tests:
+    executor: test-runner
+    steps:
+      - run: 
+          name: Setup Splunk operator and enterprise image env vars
+          command: |
+            echo 'export SPLUNK_OPERATOR_IMAGE=${IMAGE_NAME}:${CIRCLE_SHA1}' >> $BASH_ENV
+            echo 'export SPLUNK_ENTERPRISE_IMAGE=${ENTERPRISE_IMAGE_NAME}' >> $BASH_ENV
+            echo 'export COMMIT_HASH=$(echo ${CIRCLE_SHA1:0:7})' >> $BASH_ENV
+            echo 'export CLUSTER_NODES=${NUM_NODES}' >> $BASH_ENV
+            echo 'export CLUSTER_WORKERS=${NUM_WORKERS}' >> $BASH_ENV
+            echo 'export TEST_CLUSTER_KIND=eks' >> $BASH_ENV
+      - kubernetes/install
+      - aws-cli/install
+      - aws-eks/install-eksctl
+      - checkout
+      - attach_workspace:
+          name: Restore workspace
+          at: /tmp
+      # load the operator image to local registry in the VM
+      - load_image
+      - run:
+          name: Print out version and environment
+          command: |
+            ls -al
+            echo "GO VERSION=`go version`"
+            echo "CIRCLE_SHA1=$CIRCLE_SHA1"
+            echo "SPLUNK_OPERATOR_IMAGE=$SPLUNK_OPERATOR_IMAGE"
+            echo "SPLUNK_ENTEPRISE_IMAGE=$SPLUNK_ENTERPRISE_IMAGE"
+            echo "PRIVATE_REGISTRY=$PRIVATE_REGISTRY"
+            echo "CLUSTER_PROVIDER=$TEST_CLUSTER_KIND"
```

Comment:
Am I right in saying the term "KIND" in "TEST_CLUSTER_KIND" refers to the platform being used (i.e EKS vs GKE vs Kind). If yes, can we rename the variable to "TEST_CLUSTER_PLATFORM" to avoid any confusion with the Kind cluster platform?